bagitobjecttransfer/recordtransfer/static/recordtransfer/js/base/forms.js

```diff
@@ -413,12 +459,46 @@ $(() => {
     $('.add-form-row', '#transfer-form').on('click.transfer-form', (event) => {
         setupSelectOtherToggle(['.rights-select-other'], removeOther);
     })
+
+    /***************************************************************************
+     * Group Description Display
+    **************************************************************************/
+    // Add a change event listener to the group selection field
+    const selectField = $('#' + ID_SUBMISSION_GROUP_SELECTION);
+    selectField.on('change', updateGroupDescription);
+    
+    initializeGroupTransferForm();
+
+    /***************************************************************************
+     * New Group Creation Submission
+    **************************************************************************/
+    $('#submission-group-form').off('submit').on('submit', function (event) {
+        event.preventDefault();
+
+        $.ajax({
+            url: $(this).attr('action'),
+            type: $(this).attr('method'),
+            data: $(this).serialize(),
+            success: async function (response) {
+                try {
+                    handleNewGroupAdded(response.group);
+                    $('#add-new-group-dialog').dialog("close");
+                }
+                catch (error) {
+                    alert("Failed to update group options.");
```

Comment:
This error is popping up on the `/submission_group/uuid/` page when I try to make an edit to an existing group - I think this is because the ID of the form is the same in the modal and the edit form.

Reply:
The error was actually coming from the validation in `SubmissionGroupForm`. It was raising a validation error for having a group name that already exists, which happens when I try to modify an existing group by only changing the description.

I've fixed it to exclude comparing names with itself:
https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/226/commits/2078c7cb0ee06ff3cd8373e6d6a6ae63c587c235